mobile/lib/pages/single_recording_page.dart

```diff
@@ -0,0 +1,151 @@
+import 'package:audio_service/audio_service.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:radioaktywne/components/ra_playbutton.dart';
+import 'package:radioaktywne/components/ra_player/ra_player_recources.dart';
+import 'package:radioaktywne/components/utility/custom_padding_html_widget.dart';
+import 'package:radioaktywne/components/utility/ra_progress_indicator.dart';
+import 'package:radioaktywne/extensions/extensions.dart';
+import 'package:radioaktywne/models/recording_info.dart';
+import 'package:radioaktywne/resources/fetch_data.dart';
+import 'package:radioaktywne/resources/ra_links.dart';
+import 'package:radioaktywne/resources/ra_page_constraints.dart';
+import 'package:radioaktywne/state/audio_handler_cubit.dart';
+
+/// Page displaying a single recording.
+class SingleRecordingPage extends StatefulWidget {
+  const SingleRecordingPage({super.key, required this.recording});
+  final RecordingInfo recording;
+
+  @override
+  _SingleRecordingPageState createState() => _SingleRecordingPageState();
+}
+
+class _SingleRecordingPageState extends State<SingleRecordingPage> {
+  String _fullImagePath = '';
+
+  /// Space between things on the page.
+  static const SizedBox _emptySpace = SizedBox(height: 9);
+
+  /// Space from the top of the page.
+  static const SizedBox _spaceFromTop = SizedBox(height: 26);
```

Comment:
I'm pretty sure this gap should be defined in RaPageConstraints (and if it isn't - it definitely should be).